source/dox/Internals/API.cpp

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
Oops.  This was just an  experiment because doxygen won't process directives in .md files.
I did not intend to add it to this PR, but it's also stupid that 'dox' is in the 'source' directory.
I did 'git add source' and forgot that was there.